corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -352,7 +352,8 @@ def __init__(self, suite, module, detail_section_elements, endpoint_id, case_ses
         self.case_session_var = case_session_var
 
     def get_post_relevant(self):
-        return CaseClaimXpath(self.case_session_var).default_relevant()
+        if not self.module.case_details.short.multi_select:
+            return CaseClaimXpath(self.case_session_var).default_relevant()
```

Comment:
why doesn't this use `module.get_relevant`?

Reply:
`get_relevant` isn't on `Module`, it's on `CaseSearch`, and this code doesn't have a case search config.

Although I could move `get_relevant` up to module and name it something like `get_claim_relevant`, which would reduce the `module.case_details.short.multi_select` checks. I'll try that out.

Reply:
no need to change it since there is a good reason